log-service/src/main/java/com/example/soap/service/LogEntryImpl.java

```diff
@@ -24,6 +24,20 @@
         LocalDateTime fromDateTime = null;
         LocalDateTime toDateTime = null;
 
+        Log.ChangeType type = null;
```

Comment:
Is there a reason why you are still using the change type as string here?

Reply:
When i put it as ChangeType and tried it on SoapUI it didn’t work for me, that’s why I kept it as String here and then check if it’s add/update/delete and then created this: Log.ChangeType type = null;

Reply:
If you need to get the change type as string, you can implement a toString() in the enumerator in stead of doing it here, so we can reuse it the future if needed.